app/models/invite.rb

```diff
@@ -160,6 +183,38 @@ def self.create_invite_by_email(email, invited_by, opts = nil)
     invite
   end
 
+  # multiple use invite link
+  def self.generate_multiple_use_invite_link(invited_by:, max_redemptions_allowed: 5, expires_at: 1.month.from_now, group_ids: nil)
+    create_args = {
+      invited_by: invited_by,
+      max_redemptions_allowed: max_redemptions_allowed.to_i,
```

Comment:
We'll need to validate that this falls into a reasonable set of values.

Reply:
Done.